IsoLib/hevc_extractors/src/HEVCExtractorReader.cpp

```diff
@@ -384,7 +411,6 @@ std::vector<char> HEVCExtractorReader::getNextAUResolveExtractors()
               uiDataLength = uiRefSampleSize - uiDataOffset;
               bNaluLengthRewrite = true;
             }
-            else { assert(0); } // this should not happen if the file is OMAF compliant
```

Comment:
I see similar check at other place. Is it intentional to remove it ?

Reply:
I think you are right, and we should keep the assertion but remove the comment as it's not only about OMAF.